_startups/ban.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Base Adresse Nationale
+mission: "Un référentiel national ouvert : de l'adresse à la coordonnée géographique."
+domain: adresse.data.gouv.fr
+repository: https://github.com/etalab/ban
+status: consolidation
```

Comment:
https://github.com/sgmap/sgmap.github.io/blob/master/_config.yml#L29-L52

(and https://github.com/sgmap/sgmap.github.io/blob/master/CONTRIBUTING.md)